### ISSUE TITLE
Django upgrade time

### ISSUE THREAD

**Issue by bethlakshmi** [bethlakshmi/GBE2#1245]
It's time.  Django 1.8 (our version) is behind the times and just fell out of support.

I'd like to go all the way up to Django 1.11.  There's now a Django 2.0, but that sounds big, and I'll want more time to get my head around it.  However big the upgrades are from 1.8 to 1.11 they are probably worth it to stay in something supported.

I plan to look at this shortly after doing the hosting investigation work.

**Comment by bethlakshmi** [comment]
Attempt #2 - try to upgrade Django CMS w/out GBE

**Comment by bethlakshmi** [comment]
Stopping point tonight:
- Got migrations using only Django CMS to start.
- blocked on reversion having issues with old image_gallery stuff that it can't find because its not in current package.
- also had to remove the ad-rotator because it was too old and the migration didn't work.

Next things to try:
- figure out how others solved the reversion problem 
- figure out how to delete really old versions of our pages - maybe I can do this on the test site simluating the live data.
- Focus, instead on a great leap forward - install Python 3 & Django CMS as new installation and manually port our pages.  BONUS - may be able to integrate a more clean integration of our code from this point.

**Comment by burlexpo** [comment]
New ad rotator at https://github.com/razisayyed/django-ads seems to have been worked on within the last 6 months, so I hope it's compatible with current Django

**Comment by burlexpo** [comment]
This may want to be a separate issue, but I may have found a better host --
https://www.divio.com/plans/

Roughly $4/month includes 

- Test server included
- Postgres database
- Daily backups (7 scheduled and 3 manual with 7 days retention) 
- Content Delivery Network (CDN) for your media files
- 2 domains (excludes registration)
- Divio developer tools including Divio Desktop and Divio CLI
- 10 GB object storage and 1 TB transfer per month
- SSL certificate

This might be _much_ faster than you trying to do the upgrades manually...
